guides/v2.3/extension-dev-guide/declarative-schema/data-patches.md

```diff
@@ -120,6 +119,22 @@ The following code sample defines a data patch class that has a dependency.
     }
 ```
 
+## Reverting data patches
+
+Magento does not allow the reverting of a particular module data patch. It only allows the reverting of all module data patches per the `module:uninstall` command.
+
+Run the folowing command to revert all the module's data patches (only for modules that have been installed through `composer`):
+
+```bash
+bin/magento module:uninstall Vendor_ModuleName
+```
+
+To revert all `non composer` data patches, run:
```

Comment:
```suggestion
Run the following command to revert all `non-composer` installed data patches:
```

Reply:
**fixed**

Reply:
@erikmarr , request changes have been implemented. Could you please review?

Thank you!